src/Symfony/Bundle/FrameworkBundle/Tests/Translation/TranslatorTest.php

```diff
@@ -94,14 +94,14 @@ public function testTransWithCaching()
         $this->assertEquals('foobarbax (sr@latin)', $translator->trans('foobarbax'));
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     */
     public function testTransWithCachingWithInvalidLocale()
     {
         $loader = $this->getMock('Symfony\Component\Translation\Loader\LoaderInterface');
-        $translator = $this->getTranslator($loader, array('cache_dir' => $this->tmpDir), 'loader', '\Symfony\Bundle\FrameworkBundle\Tests\Translation\TranslatorWithInvalidLocale');
+        $translator = $this->getTranslator($loader, array('cache_dir' => $this->tmpDir));
         $translator->setLocale('invalid locale');
```

Comment:
I thought about moving this test into Translator component but it should be in a separate PR.